src/pages/SingleCell/styles.css

```diff
@@ -0,0 +1,616 @@
+/* styles.css */
+

```

Comment:
Due to some custom functionalities, I have written custom CSS while trying to maintain the overall composition of the cBioPortal theme. If required, it will be changed.